src/FortniteClient.php

```diff
@@ -129,54 +189,56 @@ public static function sendFortniteGetRequest($endpoint, $access_token, $extra_h
      * @param  array  $params       Request parameters, using the name as the array key and value as the array value
      * @return object               Decoded JSON response body
      */
-    public static function sendFortnitePostRequest($endpoint, $access_token, $params = null) {
+    public static function sendFortnitePostRequest($endpoint, $access_token, $params = null)
+    {
         $client = new Client();
 
         try {
-             $response = $client->post($endpoint, [
+            $response = $client->post($endpoint, [
                 'json' => $params,
                 'headers' => [
                     'User-Agent' => self::FORTNITE_USER_AGENT,
                     'Authorization' => 'bearer ' . $access_token
                 ]
             ]);
 
-            return json_decode($response->getBody()->getContents());      
-        } catch (GuzzleException$e) {
+            return json_decode($response->getBody()->getContents());
+        } catch (GuzzleException $e) {
             throw $e; //Throw exception back up to caller
         }
-
     }
 
-    public static function sendFortniteDeleteRequest($endpoint, $access_token) {
-        $client = new Client();
```

Comment:
sendFortniteDeleteRequest has no $client argument

Reply:
Fixed in https://github.com/Tustin/fortnite-php/pull/65/commits/8ee694bc8856343fa12e5b2376e383668e2fc848